src/lib/biocaml_algo_edit_distance.ml

```diff
@@ -0,0 +1,35 @@
+(** Allowed edit operations while computing edit distance. This type is for
+    internal use only, so it is not exported *)
+type edit_operation = 
+    Insert of int*char | Delete of int | Substitute of int*char;;
+
+(** Simple edit distance calculation bitween 2 strings. Only three operations: 
+    Insert, Delete and Substitue are taken into account. All the operations 
+    have the same weight equal to 1. The first and the second parameters
+    are the strings to compare.
+
+    This is a dynamic programming algorithm that requires m*n memory and
+    m*n time (m and n are the lengths of the strings to compare).
+*)
+let simpleEditDistance s t =
```

Comment:
Going to O(min(m, n)) memory is too easy not to implement it. Also it might be useful to parametrize the edit distance with a cost matrix.